deck_chores/utils.py

```diff
@@ -80,18 +80,23 @@ def trueish(value: str) -> bool:
 
 DEBUG = trueish(os.getenv('DEBUG', 'no'))
 
-
 log = logging.getLogger('deck_chores')
 log_handler = logging.StreamHandler(sys.stdout)
 log.addHandler(log_handler)
 log.setLevel(logging.DEBUG if DEBUG else logging.INFO)
 
+class ExcludeErrorsFilter(logging.Filter):
```

Comment:
the `logging` docs say, that a function can be used as filter since Python 3.2. i prefer that.